core/templates/dev/head/pages/exploration_player/AnswerClassificationService.js

```diff
@@ -111,8 +111,7 @@ oppia.factory('AnswerClassificationService', [
         var interactionIsTrainable = INTERACTION_SPECS[
           oldState.interaction.id].is_trainable;
 
-        if (ruleBasedOutcomeIsDefault && interactionIsTrainable &&
-            ENABLE_ML_CLASSIFIERS) {
```

Comment:
Just to check -- this is supposed to be removed?

Reply:
Oops, my mistake. I wanted to make sure that training data classification is allowed irrespective of ml classifiers are enabled or not.

Reply:
Ah, sorry -- so, just to confirm, is the intention to remove it or not?

I think we should have a flag to hide the training data stuff though, at least until the ML stuff is actually working. Otherwise the training data UI components will clutter the UI in a way that's somewhat opaque to creators.

Reply:
It will be removed from this line and added to a later line (where classifier is used).

Reply:
OK, got it. Thanks!